EIPS/eip-2400.md

```diff
@@ -0,0 +1,110 @@
+---
+eip: 2400
```

Comment:
```suggestion
eip: 5217
```

Reply:
Never mind, disregard this. For some reason, GitHub is showing this as a brand-new file.